README.md

```diff
@@ -1,6 +1,306 @@
 # CNS Object Broker
-## For dynmically provisioning object storage compliant with an S3 interface.
+## Utilize Kubernetes Service-Catalog to dynamically provision CNS Object Storage.
 
 ## WARNING!
+The work in this project is a proof of concept and not intended production.
 
-The work in this project is a proof of concept and not intended for anything more than that.
+## Overview
+A core feature of the Kubernetes system is the ability to provision a diverse
+offering of block and file storage on demand.
+This project seeks to demonstrate
+that by using the Kubernetes-Incubator's Service-Catalog, it is now also possible
+to bring this dynamic provisioning to S3 object storage.
+This broker is designed
+to be used with [Gluster-Kubernetes](https://github.com/jarrpa/gluster-kubernetes).
+
+See our detailed [command flow diagram](docs/diagram/control-diag.md) for
```

Comment:
Our ~~detailed~~ command flow diagram shows where Kubernetes, service-catalog, and the (often external) service broker interact._